
Experiment dir: ./exp/Test_los_west_baseline
[34m[1mwandb[39m[22m: [33mWARNING[39m No relevant files were detected in the specified directory. No code will be logged to your run.
Traceback (most recent call last):
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/scripts/../tests/test.py", line 614, in <module>
    main()
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/scripts/../tests/test.py", line 463, in main
    TestLSTM().test_standard_lstm(
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/tests/test_standard_lstm.py", line 324, in test_standard_lstm
    val_loaders = get_val_loaders(
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/utils.py", line 176, in get_val_loaders
    val_dataset = LOSDataset(
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/los/dataset.py", line 62, in __init__
    self._read_data(root, listfile)
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/los/dataset.py", line 103, in _read_data
    self.reader.read_example(0)["X"]
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/readers.py", line 241, in read_example
    (X, header) = self._read_timeseries(name, t)
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/readers.py", line 195, in _read_timeseries
    with open(os.path.join(self._dataset_dir, ts_filename), "r") as tsfile:
FileNotFoundError: [Errno 2] No such file or directory: '/data/datasets/mimic3-benchmarks/data/length-of-stay/train/4469_episode298308_timeseries_intime_2131-10-14_05-30-57_outtime_2131-10-18_18-14-37.csv'